Default collection install to first path in COLLECTIONS_PATHS (#62870)
ansible/ansible: docs/docsite/rst/dev_guide/developing_collections.rst

Trying collection locally
-------------------------

You can try your collection locally by installing it from the tarball. The following will enable an adjacent playbook to
access the collection:

.. code-block:: bash

>    ansible-galaxy collection install my_namespace-my_collection-1.0.0.tar.gz -p ./collections

Reviewer: @acozine @chouseknecht **If** this PR as a whole gets deferred to 2.10, should the above two changes be pushed into 2.9.x separately?  It seems like it fixes an independent documentation bug.

Reviewer 2: I am pretty positive that the change to give a default install path will not be accepted for 2.9, potentially not even for 2.10.

You should use one of the values configured in :ref:`COLLECTIONS_PATHS` for your path. This is also where Ansible itself will

> expect to find collections when attempting to use them. If a path value is not specified, collection will be installed to the first path

Reviewer 3: ```suggestion
expect to find collections when attempting to use them. If you don't specify a path value, ``ansible-galaxy collection`` installs the collection in the first path
```

Author: Fixed

defined in :ref:`COLLECTIONS_PATHS`, which by default is ``~/.ansible/collections``.

> Next, try using the local collection inside a playbook, for examples and more details see :ref:`Using collections <using_collections>`

Reviewer 3: ```suggestion
Next, try using the local collection inside a playbook. For examples and more details see :ref:`Using collections <using_collections>`.
```

Author: Fixed
